<ui version="4.0" >
 <class>ConfigureBase</class>
 <widget class="QDialog" name="ConfigureBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Device Updater</string>
  </property>
  <property name="windowIcon" >
   <iconset>package_green.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="serverNameLabel" >
       <property name="text" >
        <string>Server</string>
       </property>
       <property name="buddy" >
        <cstring>serverNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverNameLineEdit" >
       <property name="toolTip" >
        <string>IP address of this machine, reachable from device</string>
       </property>
       <property name="text" >
        <string>10.10.10.21</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portLabel" >
       <property name="text" >
        <string>Port</string>
       </property>
       <property name="buddy" >
        <cstring>portLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit" >
       <property name="toolTip" >
        <string>port to listen on</string>
       </property>
       <property name="text" >
        <string>8080</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Package Root Dir</string>
       </property>
       <property name="buddy" >
        <cstring>packageRootLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="packageRootLineEdit" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="packageRootDirPushButton" >
       <property name="text" >
        <string>Choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="packageDescriptorCommandLabel" >
       <property name="text" >
        <string>Descriptor command</string>
       </property>
       <property name="buddy" >
        <cstring>commandLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commandLineEdit" >
       <property name="toolTip" >
        <string>Script to run to create package descriptors</string>
       </property>
       <property name="text" >
        <string>mkPackages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="descriptorPushButton" >
       <property name="text" >
        <string>Choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
